main_routers/system_router.py

```
@@ -1912,6 +2032,40 @@ async def _llm_call_with_retry(
                     logger.info(f"[{lanlan_name}] Phase 1 音乐话题已添加: {music_topic[:100]}...")
                     phase1_topics.append(('music', music_topic))
         
+        # 表情包模式特殊处理：不经过 Phase 1 LLM 筛选，直接添加表情包话题
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)